source/dbo/Stored Procedures/Data Migration/uspMigrationOutcomeMismatch.sql

```diff
@@ -0,0 +1,38 @@
+﻿/*
+	Note that we treat some outcomes as being equal even when they are actually different in NTBS and ETS.
+	For more details see uspMigrationMisMatchedOutcomes.sql.
+*/
+
+CREATE PROCEDURE [dbo].[uspMigrationOutcomeMismatch]
+	@MigrationRun INT = NULL
+AS
+	WITH EventOrder AS
+	(SELECT 'DiagnosisMade' AS EventName, 1 AS 'OrderBy'
+		UNION
+		SELECT 'TreatmentStart' AS EventName, 2 AS 'OrderBy'
+		UNION
+		SELECT 'TransferOut' AS EventName, 3 AS 'OrderBy'
+		UNION
+		SELECT 'TransferIn' AS EventName, 4 AS 'OrderBy'
+		UNION
+		SELECT 'TransferRestart' AS Eventname, 5 AS 'OrderBy'
+		UNION
+		SELECT 'TreatmentOutcome' AS EventName, 6 AS 'OrderBy')
```

Comment:
This trick is duplicated in `uspGenerateMigrationResultsData.sql` - we could commonise the two, by joining to a view of notifications and their most recent treatment event. Do you think this would be worthwhile?

Reply:
The NTBS and ETS treatment outcomes are stored in Migration Run Results, there's no need to recalculate them here.  What I was looking for was a list of the treatment events from NTBS with their dates, as this will normally illustrate why the outcomes are mismatched.

Reply:
I thought the idea here was to include the most recent treatment event (i.e. the one from which the outcome is taken)... I didn't realise we wanted to include all the treatment events.